src/constants/tokens.ts

```diff
@@ -401,7 +401,7 @@ function isMatic(chainId: number): chainId is SupportedChainId.POLYGON | Support
 class MaticNativeCurrency extends NativeCurrency {
   public constructor(chainId: number) {
     if (!isMatic(chainId)) throw new Error('Not matic')
-    super(chainId, 18, 'MATIC', 'Polygon Matic')
+    super(chainId, 18, 'POLY', 'Polygon Matic')
```

Comment:
Matic reference still noticed (across the codebase aswell). Remove all matic references.